pkg/apis/marketplace/v1alpha1/register.go

```diff
@@ -1,41 +1,22 @@
+// NOTE: Boilerplate only.  Ignore this file.
+
+// Package v1alpha1 contains API Schema definitions for the marketplace v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=marketplace.redhat.com
 package v1alpha1
 
 import (
-	sdkK8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-)
-
-const (
-	version   = "v1alpha1"
-	groupName = "marketplace.redhat.com"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
 )
 
 var (
-	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
-	// SchemeGroupVersion is the group version used to register these objects.
-	SchemeGroupVersion      = schema.GroupVersion{Group: groupName, Version: version}
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "marketplace.redhat.com", Version: "v1alpha1"}
+
+	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
+	SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
+
 	OperatorSourceKind      = "OperatorSource"
 	CatalogSourceConfigKind = "CatalogSourceConfig"
 )
-
-func init() {
-	sdkK8sutil.AddToSDKScheme(AddToScheme)
-}
-
-// addKnownTypes adds the set of types defined in this package to the supplied scheme.
-func addKnownTypes(scheme *runtime.Scheme) error {
```

Comment:
Question: Has this been moved, or we don't need it anymore?

Reply:
It's not needed anymore.